PIM/Operation/01_ManageProducts.md

```diff
@@ -0,0 +1,304 @@
+[!!Omni-Channel](Channels)   
+[!!User Interface PIM](../UserInterface/00_UserInterface.md)   
+[!!Manage the variants](./02_ManageVariants.md)
+
+
+# Manage a product
+
+A product defines everything that can be offered and purchased on a market. All products are assigned to an attribute set and can be specified by the attributes within this set. You can create master products and [variants](./02_ManageVariants.md#create-variants), edit and copy products. To offer a product on a marketplace you have to create an offer to the product, see [Create an offer in PIM](#create-an-offer-in-PIM).
+
+## Create a product
+
+Create a product to offer it on a marketplace. You can create a product or create variants, see [Create variants](./02_ManageVariants.md#create-variants). In the following, the procedure to create a product is described.
+
+#### Prerequisites
+
+An attribute set is created, see [Create an attribute set](../Integration/02_ManageAttributeSets.md#Create-an-attribute-set).
+
+#### Procedure
+*PIM > Products > Tab LIST*
+
+![Product list](../../Assets/Screenshots/PIM/Products/List/Products.png "[Product list]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+  The *Create new product* window is displayed.
+
+  ![Create new product](../../Assets/Screenshots/PIM/Products/List/CreateNewProduct.png "[Create new product]")
+
+2. Select an attribute set in the *Attribute set* drop-down list. All active attribute sets are displayed in the drop-down list.
+
+3. Enter a SKU for the product in the *SKU* field.
+
+  > [Info] The SKU (Stock Keeping Unit) is an identification number for the product. Therefore, it should be uniquely assigned to a single product.
+
+4. Click the [CREATE] button.   
+  The *Create Product* view is displayed. The *Attributes* tab is preselected.
+
+  ![Attributes](../../Assets/Screenshots/PIM/Products/List/Attributes/AttributesCreate.png "[Attributes]")
+
+5. Enter the data for the product in the *Attributes* tab. You must at least complete the *Product Name*, *Price* and *Tax class* fields.   
+
+  > [Info] All fields marked with the note *(required)* are used for the completeness calculation of product attributes. You can enable the *show empty required attributes only* toggle to display only the incomplete required attribute fields for the selected product.
+
+6. Click the [SAVE] button in the upper right corner.   
+  The new product is saved. The *Create product* view is closed. The product list with the new product is displayed.
+
+
+
+## Edit products
+
+After you have created a product, you can edit it. You can either edit a single product or use the mass editing function to edit the same attribute values in several products using a wizard. Depending on the attribute settings, not all attribute values are editable.
```

Comment:
There are some other conditions aswell, that might make an attribute not editable:
- If you are currently editing a variant Product and the Attribute is configured to be copied down from the parent
- If the attribute is set via an ETL mapping